quantum/stm32/chconf.h

```diff
@@ -48,7 +48,7 @@
  * @details Frequency of the system timer that drives the system ticks. This
  *          setting also defines the system tick time unit.
  */
-#define CH_CFG_ST_FREQUENCY                 100000
+#define CH_CFG_ST_FREQUENCY                 1000
```

Comment:
We may not want to change this, as it will slow down the board.

Doing so cause serious issues with animations in the planck ez, actually, as well as issues with timing for keys that used any sort of timing (software or otherwise). 

```suggestion
#define CH_CFG_ST_FREQUENCY                 100000
```

Reply:
I’ll have to test to see if this messes with anything first

Reply:
hi @ishtob, i've been working on this long time ago together with @jackhumbert and this frequency change is result of my suggestion to him. At the time this arm_rgb branch was developed, this seemed like a needed change because all the animations were way to fast. I tested today and this doesn't seem to be the case anymore and you can revert this frequency change as @drashna suggested. As you might already know, i opened another PR #6516 without knowing of this PR existance (my bad, should've checked before opening new PR).

Reply:
Drashna is right about the settings here, the global system time frequency is being reduced and we're also eliminating any benefits gained from tickless mode in chibi by setting the delta to 0 (classic mode).

I'm not sure but i one of the problems in this case is the pin assignment for this board. A0 seems to only have access to TIM2_1 and some advanced features for TIM8. This implies that any usage of PWM on this pin could potentially impact the entire system due to how ST is assigned. 

Another issue I've been facing is this implementation uses every channel available on TIM3. Consequently breaking pwm for any downstream board that might use that timer for another purpose (mainly backlight).